### DISCUSSION TITLE
Why map <C-A> as select all?

### DISCUSSION THREAD

**Discussion by TonyWu20** [ayamir/nvimdots#690] [General]
Mapping select all in vim is meaningless by all means, not to mention that this robs the original feature of <C-A>: increment the number under cursor by one. Please do not overwrite this original mapping in both vim and neovim.

**Comment by charliie-dev** [comment]
Sorry, that's on me. I added those keymaps.
U can use those keymaps for now.
https://github.com/ayamir/nvimdots/blob/f3520c91feba26997a5ca22a4f92de8f21d31c06/lua/core/mapping.lua#L31-L32

**Reply by charliie-dev** [reply]
@ayamir @Jint-lzxy do u guys think we should revert those keymaps?

**Reply by ayamir** [reply]
Indeed, just keep them default is enough.

**Reply by charliie-dev** [reply]
then i shall revert those keymaps.

**Reply by ayamir** [reply]
f9ca54962b14d4be3d612229107c0bf44e68dc94
